app/views/tracks/index.html.haml

```diff
@@ -6,38 +6,84 @@
         %span.notranslate
           = @conference.title
 
+  .row
+    .col-md-12
+      %p.text-right
+        = link_to '#status-help', class: 'btn btn-default',  "data-toggle"=>"collapse" do
+          Help?
+      .collapse#status-help
+        %p
+          %strong
+            What happens next with my track request?
+        %p
+          If you submit a track request, the conference organizers will review it and either accept or reject it.
+          %br
+          If your track request is accepted, the conference organizers expect you to confirm that you will be able to hold it.
+          Then you will gain the Track organizer role.
+          %br
+          If your track request is rejected, you can either live with that or adapt it and resubmit it for review again.
+          %br
+          If something changes and you can't organize the track any more, you should withdraw it.
+
+        %p
+          %strong
+            Why do I need to add more information?
```

Comment:
This is not needed, is it? We don't ask people to add extra information, except for the fields that they completed during request creation.